rotkehlchen/db/evmtx.py

```diff
@@ -129,6 +130,17 @@ def add_evm_transactions(
             tuples=tx_tuples,
             relevant_address=relevant_address,
         )
+        authorization_list = [
+            (tx.tx_hash, tx.chain_id.serialize_for_db(), entry.nonce, entry.delegated_address)
+            for tx in evm_transactions if tx.authorization_list is not None
+            for entry in tx.authorization_list
+        ]
+        if len(authorization_list) > 0:
+            write_cursor.executemany(
+                'INSERT INTO evm_transactions_authorization(tx_hash, chain_id, nonce, delegated_address) '  # noqa: E501
+                'VALUES (?, ?, ?, ?)',
+                authorization_list,
+            )
```

Comment:
if you change the schema to have the primary key I believe you need to handle possible errors here